java/java.lsp.server/src/org/netbeans/modules/java/lsp/server/Utils.java

```diff
@@ -96,6 +101,97 @@ public static SymbolKind elementKind2SymbolKind(ElementKind kind) {
         }
     }
 
+    public static String label(CompilationInfo info, Element e, boolean fqn) {
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The cyclomatic complexity of this method is 18. By comparison, 99% of the methods in the CodeGuru reference dataset have a lower cyclomatic complexity. This indicates the method has a high number of decisions and it can make the logic difficult to understand and test. We recommend that you simplify this method or break it into multiple methods.